docs/elements/hx-tile-title/index.html

```diff
@@ -0,0 +1,29 @@
+---
+title: <hx-tile-title>
+also:
+  components/choice-tiles: Choice Tiles
+  elements/hx-tile: <hx-tile>
+  elements/hx-tile-description: <hx-tile-description>
+---
+{% extends 'element.njk' %}
+{% block content %}
+  <section>
+    <p>
+      The custom <code>&lt;hx-tile-title&gt;</code> element holds the title for
+      a tile.
+    </p>
+
+    <hx-dl class="hxBox-md metadata">
+      <hx-def>
+        <hx-dt>Permitted Parents</hx-dt>
+        <hx-dd>
+          an <a href="elements/hx-tile">&lt;hx-tile&gt;</a> element
```

Comment:
See question above.

Reply:
@CITguy do we need to update the metadatas?

Reply:
Same response as above :) Blame it on Friday brain; y'all have done it elsewhere, so I'd leave it to match the rest of the site.

Reply:
see counter-comment above :)